doc/doxygen/Makefile

```diff
@@ -40,8 +40,19 @@ src/changelog.md: src/changelog.md.tmp ../../release-notes.txt
 	@./generate-changelog.py $+ $@
 
 .PHONY:
-latex: src/changelog.md
+latex: git-lfs src/changelog.md
 	( cat riot.doxyfile ; echo "GENERATE_LATEX= yes" ) | doxygen -
 
+RED=\033[0;31m
+NC=\033[0m
+
+git-lfs: git-lfs-check git-lfs-install
+
+git-lfs-check:
+	@git lfs --version > /dev/null 2>&1 || (echo -e "$(RED)Please install git lfs.$(NC)" && exit 1)
+
+git-lfs-install:
```

Comment:
I don't think it is OK to install something when told to do another thing. Also, why should building the docs depend on git being present?

Reply:
it was the first approach for "making sure the documentation is generated OK`. But as we discussed offline, it might not be a dependency.

Reply:
I will replace this with a warning

Reply:
> I don't think it is OK to install something when told to do another thing.

Regarding this, the command name is misleading. The `git lfs install` command just enables Git LFS in that repo. The `--local` flag makes sure it's only on that repo and not globally.